### PR TITLE
FYLE-3xge43: Split expense screen

### PR DESCRIPTION
**Pull request by Dimple16**

### CODE REVIEW

**Comment by Dimple16** [comment]
Screenshot 2021-06-25 at 4 54 19 PM
Screenshot 2021-06-25 at 4 54 45 PM
Screenshot 2021-06-25 at 5 04 51 PM

File: src/app/fyle/split-expense/split-expense.page.scss [hunk 1]
```diff
@@ -13,18 +13,39 @@ $background-color: #fff;
...
+      font-weight: 500;
+      margin: 0 auto;
+    }
+
+    &-save {
+      color: $primary;
+      font-size: 16px;
+      line-height: 20px;
+      font-weight: 500;
+      text-align: right;
```
Comment (iamtarun1993) [inline src/app/fyle/split-expense/split-expense.page.scss:37]: use text-right class

File: src/app/fyle/split-expense/split-expense.page.scss [hunk 2]
```diff
@@ -34,18 +55,25 @@ $background-color: #fff;
   }
 
   &--internal-block {
-    padding: 16px 0;
+    padding: 14px 0;
     background: #fff;
```
Comment (iamtarun1993) [inline src/app/fyle/split-expense/split-expense.page.scss:59]: use variable

File: src/app/fyle/split-expense/split-expense.page.scss [hunk 3]
```diff
@@ -34,18 +55,25 @@ $background-color: #fff;
...
     border: 0;
     line-height: 1;
-    width: 50%;
+    width: 48%;
+    height: 80px;
   }
 
   &--message {
     font-size: 16px;
     color: #ababab;
```
Comment (iamtarun1993) [inline src/app/fyle/split-expense/split-expense.page.scss:68]: same here

**Review by Julias0 — APPROVED** [review]

**Pull request merged**